shell/security/prismaci/twist-scan.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
```

Comment:
Use bash if possible

```suggestion
#!/usr/bin/env bash
```